pandas/core/indexes/datetimes.py

```diff
@@ -1112,9 +1113,11 @@ def slice_indexer(self, start=None, end=None, step=None, kind=None):
     _is_monotonic_decreasing = Index.is_monotonic_decreasing
     _is_unique = Index.is_unique
 
-    _timezone = cache_readonly(DatetimeArray._timezone.fget)
-    is_normalized = cache_readonly(DatetimeArray.is_normalized.fget)
-    _resolution = cache_readonly(DatetimeArray._resolution.fget)
+    _timezone = cache_readonly(DatetimeArray._timezone.fget)  # type: ignore
+    is_normalized = cache_readonly(
+        DatetimeArray.is_normalized.fget)  # type: ignore
+    _resolution = cache_readonly(
+        DatetimeArray._resolution.fget)  # type: ignore
```

Comment:
There doesn't seems to be proper solution for `.fget` and `.fset` so ignoring it.

Reply:
Yea we had a similar conversation here:

https://github.com/pandas-dev/pandas/pull/26280#pullrequestreview-233722590

So I think OK for now though would love to find a better solution if you see anything else out there.

cc @jbrockmendel

Reply:
I have subscribed to https://github.com/python/mypy/issues/6185 If some progress is made there I will get notified.